feat: tidy shortcut help listings

microbit-matt-hillsdon opened the pull request:
- Remove outdated, duplicate and work-in-progress/not-yet-useful actions.
- Tweak styling as we drop down to three columns.
- Drive order from the category listing and reorder.

I also think we should consider removing 'e' as Escape works consistently (e.g. to close colour picker and other editors) but 'e' does not. Unsure of its original motivation though so will raise separately for discussion.

I'm planning to use this for user testing but I think we should consider merging it here too. I think likely we'll try to replace this dialog with one that is specific to MakeCode in the longer term.

Related to #65

microbit-matt-hillsdon committed:
feat: tidy shortcut help listings

- Remove duplicate and work-in-progress/not-yet-useful actions.
- Tweak styling as we drop down to three columns.
- Drive order from the category listing and reorder.

microbit-matt-hillsdon commented on src/constants.ts:
```diff
@@ -78,41 +78,37 @@ export enum MODIFIER_KEY {
+export const SHORTCUT_CATEGORIES: Record<
+  string,
+  // Also allow undo/redo. Document the non-keyboard-nav versions of others for better text
+  // because temporarily the name in the table is derived from these id-like names.
```
This was a surprise! I looked for the "Toggle keyboard nav" text to improve it and was surprised it didn't exist as such. Hopefully #155 can address this.

cpcallen replied:
This entire file would have gone away quite a long time ago except that that refactor has never been as important as making functional changes, but you are correct to suspect that we'll likely be changing how all of these shortcuts are defined at some point in the near future.

(In the mean time, though: may I encourage you to try to keep comment lines to <= 80 chars, please?)

microbit-matt-hillsdon replied:
Sure, 88374cefdd8cb8b663901f1d742d74c88c22c5b8

microbit-matt-hillsdon committed:
chore: wrap comment

microbit-matt-hillsdon committed:
fix: document Enter

It's hard to have perfect text for this as it's just the primary action and
what it does is contextual. Reviewed this suggestion with @kmcnaught.

microbit-matt-hillsdon committed:
fix: resequence based on feedback; drop toggle action

The toggle action is confusing for user testing. We'll raise it for discussion.
